.gitignore

```diff
@@ -30,3 +30,7 @@ node_modules
 
 # Webstorm ide files
 .idea
+
+# Example bundles
```

Comment:
On a second note, I think you can generate them in `npm preinstall` but we can leave that for later.